crypto/rsa/rsa_impl.c

```diff
@@ -644,11 +635,10 @@ static int mod_exp(BIGNUM *r0, const BIGNUM *I, RSA *rsa, BN_CTX *ctx) {
     goto err;
   }
 
-  /* Turn BN_FLG_CONSTTIME flag on before division operation */
-  pr1 = &local_r1;
-  BN_with_flags(pr1, r1, BN_FLG_CONSTTIME);
+  /* Ensure BN_FLG_CONSTTIME flag is on before division operation */
```

Comment:
All of the comment of this form are not needed any more because they are replaced by the assertions, so please remove all of them.